charts/cinder-csi-plugin/values.yaml

```diff
@@ -75,3 +75,12 @@ storageClass:
   retain:
     isDefault: false
     allowVolumeExpansion: true
+
+priorityClassName:
+  controllerPlugin: ""
+  nodePlugin: ""
+
+podSecurityPolicy:
```

Comment:
Please add some comments to those options (especially for `existingPolicyName` and `create `) here.

Reply:
Added comments in the latest commit